examples/todo/src/__tests__/acceptance/todo.acceptance.ts

```diff
@@ -209,17 +180,12 @@ describe('TodoApplication', () => {
       connector: 'memory',
     });
 
-    // Override Geocoder datasource to use a caching proxy to speed up tests.
-    app
-      .bind('datasources.config.geocoder')
-      .to(getProxiedGeoCoderConfig(cachingProxy));
-
     // Start Application
     await app.start();
   }
 
   async function givenTodoRepository() {
-    todoRepo = await app.getRepository(TodoRepository);
+    todoRepo = await app.get<TodoRepository>('repositories.TodoRepository');
```

Comment:
We don't have a repository class we could use here (`TodoRepository` is just a type alias for `EntityCrudRepository<Todo, number>`.